.agents/skills/typescript-expert/scripts/ts_diagnostic.py

```
@@ -0,0 +1,208 @@
+import sys
+import os
+import json
+from pathlib import Path
+
+import shlex
+
+def run_cmd(cmd: str) -> str:
+    """Run shell command and return output."""
+    try:
+        args = shlex.split(cmd)
+        result = subprocess.run(args, shell=False, capture_output=True, text=True)
```

Ruff (0.15.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)